bk-register/ROOTFS/opt/sage/beekeeper/register/register.py

```diff
@@ -33,6 +34,7 @@
 CA_FILE = os.path.join(BASE_KEY_DIR, "certca/sage_beekeeper_ca")
 USER_SERVER = "http://bk-sshd"
 
+BEEKEEPER_DB_API = "http://bk-nodes:5000"
```

Comment:
just checking .. you reference port 5000 here.  in the docker-compose you expose 5000 (for debug) which I assume you will remove at some point.

in order for this to work does bk-register and bk-nodes need to be in the same docker network?

Reply:
I introduced an env variable BEEKEEPER_DB_API . Should we do the same for USER_SERVER ?

Reply:
Port 5000 was only open for debugging purposes. I renamed USER_SERVER and made it configurable via environment variable. Now bk-register and bk-nodes do not need to be inside of a docker network, but bk-register needs to reach bk-nodes somehow.